src/routes/LoginRoute/LoginRoute.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import LoginForm from '@/components/LoginForm/LoginForm';
+
+const LoginRoute = () => {
+  return <LoginForm />;
```

Comment:
Probably files like that one don't have any sense but let's leave it as is for now.